apps/ledger-live-desktop/src/renderer/families/hedera/ExplorerLink.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import {Account, Operation} from "@ledgerhq/types-live";
+import { getTransactionExplorer, getDefaultExplorerView } from "@ledgerhq/live-common/explorers";
+import Box from "~/renderer/components/Box";
+import LinkWithExternalIcon from "~/renderer/components/LinkWithExternalIcon";
+import { t } from "i18next";
+import { openURL } from "~/renderer/linking";
+
+
+const ExplorerLink: React.FC<{ account: Account, url: string, currencyName: string, operation: Operation }> = ({ account, currencyName, operation }) => {
+    console.log("in hedera?");
+    const url = getTransactionExplorer(getDefaultExplorerView(account.currency), operation.extra.consensusTimeStamp)
```

Comment:
Now I think about it, since the only part you need to customize is the URL, and the rest of the component seems to be the same, instead of customizing the whole `ExplorerLink` component, you can just customize the `getTransactionExplorer` function (i.e add it to `LLDCoinFamily` instead of `ExplorerLink`).

Reply:
Sounds good, got it done